src/Parsers/tests/gtest_Parser.cpp

```diff
@@ -157,7 +157,7 @@ SOURCE(CLICKHOUSE(HOST 'localhost' PORT tcpPort() TABLE 'source_table'))
 LAYOUT(FLAT())
 LIFETIME(MIN 0 MAX 1000)
 COMMENT 'Test dictionary with comment';
-)sql",
+sql",
```

Comment:
isn't it a unnecessary change?

Reply:
> isn't it a unnecessary change?

it is a unit test fail, so I fix it by the way.

Reply:
> > isn't it a unnecessary change?
> 
> it is a unit test fail, so I fix it by the way.


No, it does not matter. What you changed is C++ Raw String, the `)` you  removed is part for spliting content and delimiters.  Althought I don't know why there is no compiler warning/error when you removed it.

A helpful document [string_literal](https://en.cppreference.com/w/cpp/language/string_literal)

Reply:
And the unit test failed really strange. But it didn't occur in recently master. So, it's not necessary to change this test. maybe you can try to merge master again.

Reply:
> And the unit test failed really strange. But it dose not occur in recently master. So, it's not necessary to change this test. maybe you can try to merge master again.

That's wired, I change the code and the test passes. Maybe need deep the reason.

Reply:
I think this is correctly fix for this ut.
https://github.com/ClickHouse/ClickHouse/commit/bb521cd363f56c040cdbba523e48bc8496956c62#diff-29a9dc03cc52b629c442735c065033c77fa61e1ae7504191adad7f5ef47cdce2R188

Reply:
I rollback this code.